hack/build-release.sh

```diff
@@ -0,0 +1 @@
+#place holder to minimize patching of hack/release.sh from origin
```

Comment:
It's simply because the Jenkins job "template" we have expects them.  I simply followed the example from the metrics repo for now.  I can remove these files before we merge.